x-pack/test/saved_object_api_integration/security_and_spaces/apis/resolve_import_errors.ts

```diff
@@ -67,7 +65,7 @@ export default function({ getService }: TestInvoker) {
           },
           unknownType: {
             statusCode: 403,
-            response: expectRbacForbiddenWithUnknownType,
+            response: expectRbacForbidden,
```

Comment:
Question: there weren't any changes to the tests themselves, but we're now getting back different 403s, are we filtering out the unsupported types via this route now which is why we aren't getting the old 403s?

Reply:
Correct, they are filtered out. The code before hitting bulkCreate filters out unsupported types. They are returned inside the `errors` object in the response automatically.